rholang-tutorial/yarn.lock

```diff
@@ -0,0 +1,8682 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Does this need to be checked in, or can it be generated with yarn build?

Reply:
Yes, this is a file that locks in certain versions of dependencies. From the [Yarn docs](https://yarnpkg.com/lang/en/docs/yarn-lock/#toc-check-into-source-control):

> All yarn.lock files should be checked into source control (e.g. git or mercurial). This allows Yarn to install the same exact dependency tree across all machines, whether it be your coworker’s laptop or a CI server.